phpBB/phpbb/di/container_builder.php

```diff
@@ -548,12 +548,12 @@ protected function inject_dbal_driver()
 		}
 
 		$config_data = $this->config_php_file->get_all();
-		if (!empty($config_data))
+		/*if (!empty($config_data))
 		{
 			if ($this->dbal_connection === null)
 			{
 				$dbal_driver_class = $this->config_php_file->convert_30_dbms_to_31($this->config_php_file->get('dbms'));
-				/** @var \phpbb\db\driver\driver_interface $dbal_connection */
+				/** @var \phpbb\db\driver\driver_interface $dbal_connection * /
```

Comment:
This commented line within an already commented block could lead to a 
`Parse error: syntax error, unexpected '*' ` IMHO

Reply:
It certainly could, hence the WIP tag on this ticket.